src/tests/ftest/osa/osa_online_extend.py

```diff
@@ -150,16 +88,15 @@ def run_online_extend_test(self, num_pool):
         # Create a pool
         pool = {}
         pool_uuid = []
-        total_servers = len(self.hostlist_servers)
 
-        # Extend one of the ranks (or server)
-        # rank index starts from zero
-        rank = total_servers
+        # Extend one of the ranks 4 and 5
```

Comment:
Minor: may be it should be
`Extend ranks 4 and 5`

Reply:
Sure will do.